Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.20rc2-alpine3.17
+
+LABEL maintainer="manujurado1@correo.gur.es"
+
+WORKDIR /app/test
+
+RUN adduser --disabled-password --uid 1001 test && chown test:test /app/test

```

Comment:
¿Has comprobado si la imagen base ya tiene un usuario o si tienes que crear uno? Aparte, el nombre resulta un poco confuso.

Reply:
Si, lo comprobé a la hora de hacerlo y hace falta crearlo. Respecto al nombre, que yo sepa Linux no tiene unas mejores prácticas en cuanto al naming de los usuarios, y como el usuario se ha creado para lanzar los test, le puse ese nombre.

Reply:
Sí que las hay, pero bueno, aunque resulte me resulte un poco confuso no está mal

Reply:
Parece que has usado el mismo ejemplo que otro compañero de clase. Te digo lo mismo que a él. Si no sabes qué hace `disabled-password`, no lo uses. Usad sólo lo que sepáis qué hace, y si sirve para lo que vais a usar la imagen.

Reply:
Añadí esa opción porque cuando probé al principio me dió un error de password, miré las opciones que tenía adduser y vi esa opción, la probé y me funcionó. Pero si esa opción no es adecuada miraré otra forma de solucionarlo

Reply:
Si te ha dado un problema, no has abierto un issue para resolverlo, y por lo tanto, es imposible saber si esto ha resuelto el problema o no y qué problema resuelve. Es imposible saber si es adecuada o no.

Reply:
Toda la razón del mundo. Eliminaré esa opción y plasmaré en un issue el problema para que la solución quede trazada.

Reply:
De camino, aclara por qué usas un UID 1001.